.eslintrc.js

```diff
@@ -1,12 +1,6 @@
 module.exports = {
   plugins: ["import"],
-  extends: [
-    "plugin:import/recommended",
```

Comment:
I think its because I was using a different node version. We probably don't need to fix this. I have tried with v14 (as specified here in the our package file). I think I was trying to use v17. Will revert this change.

Reply:
This is the problem for me (even with v14)

```
Creating an optimized production build...
Failed to compile.

Plugin "import" was conflicted between ".eslintrc.js" and ".eslintrc.js » plugin:import/recommended".

```